app/assets/javascripts/discourse/app/initializers/handle-cookies.js

```diff
@@ -0,0 +1,15 @@
+import { extendThemeCookie } from "discourse/lib/theme-selector";
+import { extendColorSchemeCookies } from "discourse/lib/color-scheme-picker";
+import { later } from "@ember/runloop";
+
+export default {
+  name: "handle-cookies",
+
+  initialize() {
+    // No need to block boot for this housekeeping - we can defer it a few seconds
+    later(() => {
+      extendThemeCookie();
+      extendColorSchemeCookies();
+    }, 5000);
```

Comment:
This won't cause issues in the test env? i.e. interfering with any cookie-related tests?

Reply:
I don't think it would break anything, but the 5s timer would extend beyond the end of the tests 🤦 

Added a commit to fix - thanks!